### PR TITLE
Update installer 0.14.0

### PR DESCRIPTION
**Pull request by tijcolem**
Updates for the installer for the 0.14.0. release. 

Installers have been posted to. http://urbanopt-cli-installers.s3-website-us-west-2.amazonaws.com/?prefix=0.14.0/

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -89,21 +89,21 @@ option(BUILD_PACKAGE "Build package" OFF)
 # need to  update the MD5sum for each platform and url below
 if(UNIX)
    if(APPLE)
-     set(URBANOPT_CLI_GEMS_ZIP_FILENAME "urbanopt-cli-gems-20240709-darwin.tar.gz")
-     set(URBANOPT_CLI_GEMS_ZIP_EXPECTED_MD5 "b683bb30457904041567079ca4ff0fef")
+     set(URBANOPT_CLI_GEMS_ZIP_FILENAME "urbanopt-cli-gems-20241118-darwin.tar.gz")
+     set(URBANOPT_CLI_GEMS_ZIP_EXPECTED_MD5 "8c4d1ff5b4eb0592c16b4b2d259db6cd")
    else()
      if (ARCH MATCHES "arm64")
        set(URBANOPT_CLI_GEMS_ZIP_FILENAME "urbanopt-cli-gems-20240524-linux-arm.tar.gz")
```
Comment (vtnate) [inline CMakeLists.txt:96]: Do we not need a new version for Arm?

**Review by vtnate — COMMENTED** [review]
Looks good. What about Linux Arm?

**Comment by tijcolem** [comment]
sorry, yes. let me post linux arm packages. I'll update the s3 once available.

**Review by vtnate — APPROVED** [review]
☑️

**Pull request merged**